book/src/dev/rfcs/0012-value-pools.md

```diff
@@ -72,38 +74,51 @@ https://zips.z.cash/zip-0209#specification
 
 ### Transparent Transaction Value Pool & Remaining Value
 
-Transparent inputs to a transaction insert value into a transparent transaction value pool associated with the transaction, and transparent outputs remove value from this pool. As in Bitcoin, the remaining value in the pool is available to miners as a fee.
+The sum of unspent *inputs* to the transaction: the *negation* of the sum of the transaction value balances.
+
+Consensus rules:
+
+Transparent inputs to a transaction insert value into a transparent transaction value pool associated with the transaction, and transparent outputs remove value from this pool.
 
-Consensus rule: The remaining value in the transparent transaction value pool MUST be nonnegative.
+As in Bitcoin, the remaining value in the transparent transaction value pool of a non-coinbase transaction is available to miners as a fee.
+The remaining value in the transparent transaction value pool of a coinbase transaction is destroyed.
```

Comment:
These rules were updated recently in response to our design questions, I copied the updates here.